Code/Plugin.cs

```diff
@@ -29,20 +38,24 @@ private void Awake()
 			apHandler = new APHandler();
 			apCommandHandler = new APCommand();
 			apCommandHandler.AddCommands();
-			DebugMenuManager.LogToConsole("To connect to an Archipelago server, use 'ap /connect {server:port} {slot} {password}");
```

Comment:
Re-add this, but mention instead something along the lines of "To send commands or chat to the Archipelago server, use the "ap" command followed by the text you want to send."